packages/google_adsense/test/adsense_web_test.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Run locally with flutter test --platform=chrome
+
+@TestOn('chrome')
```

Comment:
This is a good test, but if you want to, you can move it to the `example/integration_test`, so all tests stay together.

What I do in these cases, is I create a bogus "vm" test that tells users that "tests live elsewhere", see:

* https://github.com/flutter/packages/tree/main/packages/google_sign_in/google_sign_in_web/test

Reply:
Move all tests to integrations tests
Use vm tests to check for throwing